backend/app/api/crypto_paper.py

```
@@ -0,0 +1,958 @@
+                for txn in transactions:
+                    if txn.timestamp.date() <= current_date:
+                        if txn.transaction_type in [CryptoTransactionType.BUY, CryptoTransactionType.TRANSFER_IN]:
+                            # Add to value
+                            try:
+                                price_data = await price_fetcher.fetch_historical_prices(
+                                    txn.symbol, current_date, current_date, is_crypto=True
+                                )
+                                if price_data:
+                                    current_price = price_data[0]["close"]
+                                    portfolio_value += txn.quantity * current_price
+                            except:
```

Ruff (0.13.3)

Do not use bare `except`

(E722)